scenes/ui_elements/dialogue/components/balloon.gd

```diff
@@ -211,4 +222,40 @@ func _on_balloon_gui_input(event: InputEvent) -> void:
 func _on_responses_menu_response_selected(response: DialogueResponse) -> void:
 	next(response.next_id)
 
+
 #endregion
+
+
+func _on_spoke(_letter: String, _index: int, _speed: float) -> void:
+	if _has_played_sfx:
+		return
+
+	_has_played_sfx = true
+
+	if talk_sound_player.stream != SFX_MAP["pencil"]:
+		talk_sound_player.stream = SFX_MAP["pencil"]
```

Comment:
There is a single stream (the pencil SFX), so it makes more sense to put it directly in the `TalkSoundPlayer` node.